apps/frontend/src/app/Database/DataManagers/BuildSettingData.ts

```diff
@@ -62,17 +64,21 @@ export class BuildSettingDataManager extends DataManager<CharacterKey, "buildSet
     if (!optimizationTarget || !Array.isArray(optimizationTarget)) optimizationTarget = undefined
     if (typeof mainStatAssumptionLevel !== "number" || mainStatAssumptionLevel < 0 || mainStatAssumptionLevel > 20)
       mainStatAssumptionLevel = 0
+
+    if (!artExclusion || !Array.isArray(artExclusion)) artExclusion = []
+    else artExclusion = [...(new Set(artExclusion))].filter(id => this.database.arts.keys.includes(id))
```

Comment:
Wasn't there a past issue where artifact IDs would get jumbled on import of a scan? I can't replicate it now, so not sure if it is still an issue. But if so, this would cause different artifacts to be excluded if the IDs are changed

Reply:
oh right i need to add this to the import list to migrate